service-worker.js

```diff
@@ -0,0 +1,19 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+var swOptions = {
+  debug: true
+}
+
+var preCachedRessources = [
+  '{{ "/css/main.css" | prepend: site.baseurl }}?t={{ site.time | date_to_xmlschema }}"',
+  {% for page in site.pages %}
+  '{{ page.url }}',
+  {% endfor %}
```

Comment:
Since the strategy is `cacheFirst` I removed the file `main.css` which has a time in it.